libcnb/src/build.rs

```diff
@@ -27,6 +19,21 @@ pub struct BuildContext<B: Buildpack + ?Sized> {
     pub buildpack_descriptor: BuildpackToml<B::Metadata>,
 }
 
+impl<B: Buildpack + ?Sized> BuildContext<B> {
+    pub fn handle_layer<L: Layer<Buildpack = B>>(
```

Comment:
Does this need docs/example given it's part of the API that will be commonly used by buildpack authors?

Reply:
It really does! I will add some.

Reply:
Fixed in b6b8a10d81ef0e501f38